M2131 solution updates

thbanasi opened the pull request:
## Before submitting this PR please ensure that you have read the following sections and then completed the template below:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures, there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

## After the submission has been made, please look at the Validation Checks:

> Check that the validations are passing and address any issues that are present. Let us know if you have tried fixing and need help.

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

## PR Template

-----------------------------------------------------------------------------------------------------------
 **Description for the PR:**
  (Enter the description below)


**Testing Completed:**
  Yes/ No : 


-----------------------------------------------------------------------------------------------------------

shainw commented on Solutions/MaturityModelForEventLogManagementM2131/Workbooks/MaturityModelForEventLogManagement_M2131.json:
```diff
@@ -1174,7 +1209,7 @@
             "type": 1,
             "content": {
-              "json": "### Pricing\r\n✳️[Microsoft Sentinel: Ingestion](https://azure.microsoft.com/pricing/details/azure-sentinel/#pricing)<br>\r\n✳️[Log Analytics: Log Retention](https://azure.microsoft.com/pricing/details/monitor/#pricing)"
+              "json": "### Pricing\r\n✳️ [Microsoft Sentinel: Ingestion](https://azure.microsoft.com/pricing/details/azure-sentinel/#pricing)<br>\r\n✳️ [Log Analytics: Log Retention](https://azure.microsoft.com/pricing/details/monitor/#pricing)"
```
I noticed you removed some of the icons in other parts, did you want to remove here also or not?

shainw replied:
Chatted with TJ - this is by design.  Closing ocmment.

shainw reviewed (approved):
Assuming you left some of the icons in the Workbook, should be good here.

thbanasi commented:
@Shain Wray ***@***.***>,

Thanks so much for the update and for your support. Icons don’t display in color anymore in panel titles so I removed them from the panel titles in the Assessment section. In the Log Management Section I added a space after the icons for visual effect, but agreed I do want to keep them there.

Very Respectfully,

TJ Banasik
CISSP-ISSEP,ISSAP,ISSMP
Senior Program Manager
C&AI, Sovereign Cloud Security Group
***@***.******@***.***>
(202) 658-8148



***@***.***

LinkedIn: In/tjbanasik<http://www.linkedin.com/in/tjbanasik>

From: Shain ***@***.***>
Sent: Wednesday, January 5, 2022 12:00 PM
To: Azure/Azure-Sentinel ***@***.***>
Cc: TJ Banasik ***@***.***>; Author ***@***.***>
Subject: Re: [Azure/Azure-Sentinel] M2131 solution updates (PR #3842)


@shainw approved this pull request.

Assuming you left some of the icons in the Workbook, should be good here.

________________________________

In Solutions/MaturityModelForEventLogManagementM2131/Workbooks/MaturityModelForEventLogManagement_M2131.json<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2FAzure-Sentinel%2Fpull%2F3842%23discussion_r778984090&data=04%7C01%7CThomas.Banasik%40microsoft.com%7Ccee43d5448cf4e9696f108d9d06cd386%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637769988104588472%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=BsAPLkevz0qZ7EbaGa3AuL9skLv1N%2FBEWlWqwygKT70%3D&reserved=0>:

> @@ -1174,7 +1209,7 @@



           {



             "type": 1,



             "content": {



-              "json": "### Pricing\r\n✳️[Microsoft Sentinel: Ingestion](https://azure.microsoft.com/pricing/details/azure-sentinel/#pricing)<br>\r\n✳️[Log Analytics: Log Retention](https://azure.microsoft.com/pricing/details/monitor/#pricing)"



+              "json": "### Pricing\r\n✳️ [Microsoft Sentinel: Ingestion](https://azure.microsoft.com/pricing/details/azure-sentinel/#pricing)<br>\r\n✳️ [Log Analytics: Log Retention](https://azure.microsoft.com/pricing/details/monitor/#pricing)"



I noticed you removed some of the icons in other parts, did you want to remove here also or not?

—
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2FAzure-Sentinel%2Fpull%2F3842%23pullrequestreview-844839340&data=04%7C01%7CThomas.Banasik%40microsoft.com%7Ccee43d5448cf4e9696f108d9d06cd386%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637769988104588472%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=trqnPb19Jd6KcdvHq3U3NQDlRrzihra9Kt5s32CuRwg%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAM6PREQ5FJHKYJJRKN3SQUDUUR2JNANCNFSM5LKHFR3A&data=04%7C01%7CThomas.Banasik%40microsoft.com%7Ccee43d5448cf4e9696f108d9d06cd386%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637769988104588472%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=CBV%2BTU5CE90ldJ03ZbdhEWDCBPykEZdDX3G%2BwUZZGEI%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7CThomas.Banasik%40microsoft.com%7Ccee43d5448cf4e9696f108d9d06cd386%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637769988104638463%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=n8tDopA0Upznd%2BKjarNkTb2QnFq0i6RKNDmzGkalZdc%3D&reserved=0> or Android<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7CThomas.Banasik%40microsoft.com%7Ccee43d5448cf4e9696f108d9d06cd386%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637769988104638463%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=GZVMCXgRfooaGUX1HquV3Vc0Z%2BNf8hB0t0VmIvPpu7w%3D&reserved=0>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>
